easybuild/easyconfigs/b/Bowtie/Bowtie-1.1.2-foss-2015b.eb

```diff
@@ -0,0 +1,19 @@
+#Modified from existing version by:
+#Robert Schmidt <rjeschmi@gmail.com>
+#Ottawa Hospital Research Institute - Bioinformatics Team
+name = 'Bowtie'
+version = '1.1.2'
+
+homepage = 'http://bowtie-bio.sourceforge.net/index.shtml'
+description = """Bowtie is an ultrafast, memory-efficient short read aligner.
+It aligns short DNA sequences (reads) to the human genome.
+"""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+#toolchainopts = {'optarch': True, 'pic': True}
```

Comment:
Only commented them out due to my ignorance about what they are/do and the fact I don't think I've done this when previously building bowtie from source.

In fact the only thing I usually do is to run `make WITH_TBB=1` to include support for multithreading as per http://bowtie-bio.sourceforge.net/manual.shtml#building-from-source